extension/docs/DEV.md

```diff
@@ -36,16 +37,27 @@ cp .env.example .env.development
 cp .env.example .env.production
 ```
 
-Build Bergamot Translator for WASM:
+## Building Bergamot Translator WASM resources and importing them to the extension
+
+1. Make sure all submodules are initiated:
+
+```bash
+git submodule update --init --recursive
+```
+
+2. Put the model files to be included in the build into `../bergamot-translator/models/` according to the instructions [here](../bergamot-translator/README.md).
+
+3. Run the build and import script (in the `extension/` folder in this repo):
 
 ```bash
-source /path/to/emsdk_env.sh
 ./build-and-import-bergamot-translator.sh
 ```
 
-Repeat this last step any time there has been an update in the bergamot-translator submodule.
+Repeat this process any time there has been an update in the bergamot-translator submodule.
+
+Note that changing the files in the `../bergamot-translator/models/` folder doesn't automatically invalidate the build, so better remove `../bergamot-translator/build-wasm-docker/` in those cases in order to trigger a full rebuild.
```

Comment:
@motin It would be faster if we just remove `../bergamot-translator/build-wasm-docker/wasm` folder. It will just invoke  rebuilding the file packaging step and not do a full re-compilation.

Reply:
Great that you found that! Updating docs...